lua/tree-sitter-go-template/init.lua

```diff
@@ -0,0 +1,87 @@
+local M = {}
+
+local lang_name = "gotmpl"
+
+local known_additional_langs = {
+  "helm",
+}
+
+local path = debug.getinfo(1).source:sub(2):match "(.*[/\\])" or "./"
+local parser_config = require("nvim-treesitter.parsers").get_parser_configs()
+
+--- @param additional_langs string[]
+local function configure_filetype(additional_langs)
+  vim.filetype.add {
+    extension = {
+      [lang_name] = lang_name
+    },
+  }
+
+  if vim.tbl_contains(additional_langs, "helm") then
+    vim.filetype.add {
```

Comment:
If we add the filetype to this plugin we maybe also want to set the commentstring, so we have everything that the vim-helm plugin has: https://github.com/towolf/vim-helm/blob/master/ftdetect/helm.vim#L4

Reply:
Also, what about a filetype for gotmpl?

Reply:
> Also, what about a filetype for gotmpl?

That is already set up in line 14.

And agreed, I will have a look at setting up the commentstring for both file types.